modules/editor/editor.js

```diff
@@ -1,8 +1,45 @@
+import InserterButton from './inserter/button';
+
+const h = wp.element.createElement;
+
+const EditorComponent = ( { content, state: { inserter }, toggleInserter } ) => {
+	return h( 'div', {},
```

Comment:
There might be a minuscule performance benefit to passing `null` instead of `{}` because it can be skipped more easily by a simple truthy test.